modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -890,3 +890,25 @@ storage:
           master=bond0
           slave-type=bond
 ----
+=== Disabling Automatic Configuration of Ethernet Devices
+
+By default, FCOS will attempt to autoconfigure (DHCP/SLAAC) on every interface with a cable plugged in. In some network environments this may not be desirable. It's possible to change this behavior of NetworkManager with a configuration file dropin:
+
+
+.Disable NetworkManager autoconfiguration of ethernet devices
+[source, yaml]
+----
+variant: fcos
+version: 1.4.0
+storage:
+  files:
+     - path: /etc/NetworkManager/conf.d/noauto.conf
+      mode: 0644 
+      contents: 
+        inline: |
+          [main]
+          # Do not do automatic (DHCP/SLAAC) configuration on ethernet devices
+          # with no other matching connections.
+          no-auto-default=* 
+----
+NOTE: WARNING: If NetworkManager autoconfiguration of ethernet devices is disabled and no other network configuration is provided the system will boot without network access.
```

Comment:
```suggestion
WARNING: If NetworkManager autoconfiguration of ethernet devices is disabled and no other network configuration is provided the system will boot without network access.
```

`WARNING:` has it's own meaning in asciidoc so let's remove the `NOTE:` in the front.